### PR TITLE
bump version to 1.0.0-a.31

### PR DESCRIPTION
**Pull request by youwen5**

### CODE REVIEW

**Comment by Airradda** [comment]
Beat me to it. Did you mean to include the `result` symlink?

File: flake.nix [hunk 1]
```diff
@@ -5,85 +5,124 @@
...
-        stdenv.cc.cc fontconfig libxkbcommon zlib freetype
-        gtk3 libxml2 dbus xcb-util-cursor alsa-lib libpulseaudio pango atk cairo gdk-pixbuf glib
-	udev libva mesa libnotify cups pciutils
-	ffmpeg libglvnd pipewire
-      ] ++ (with pkgs.xorg; [
-        libxcb libX11 libXcursor libXrandr libXi libXext libXcomposite libXdamage
-	libXfixes libXScrnSaver
+    runtimeLibs = with pkgs;
+      [
+        libGL
```
Comment (MarceColl) [inline flake.nix]: Please do not add formatting changes to this

File: result [hunk 2]
```diff
@@ -0,0 +1 @@
+/nix/store/wvjl222v0kkylwf9x7g0h07q5q1wqxcd-zen-browser-1.0.0-a.31
```
Comment (MarceColl) [inline result]: Remove this symlink

**Review by MarceColl — CHANGES_REQUESTED** [review]

**Comment by youwen5** [comment]
Whoops, didn't even realize that wasn't gitignored. Will fix in a sec

**Comment by MarceColl** [comment]
> Whoops, didn't even realize that wasn't gitignored. Will fix in a sec

Feel free to add it to gitignore! Thanks for your contribution

**Comment by youwen5** [comment]
Done, created and populated .gitignore and reverted formatting changes. I forgot my editor automatically formats everything with alejandra. Should we add an explicit `formatter` so `nix fmt` can take care of everything?

**Pull request merged**